repository-module/src/main/java/ru/mail/dimaushenko/repository/impl/ItemRepositoryImpl.java

```diff
@@ -0,0 +1,201 @@
+        }
+    }
+
+    @Override
+    public List<Item> getAllEntities(Connection connection) throws SQLException {
+        try (PreparedStatement preparedStatement = connection.prepareCall(requestProperties.getSqlRequestSelectAllItems())) {
+            try (ResultSet resultSet = preparedStatement.executeQuery()) {
+                List<Item> items = new ArrayList<>();
+                while (resultSet.next()) {
+                    Item item = getItem(resultSet);
+                    items.add(item);
+                }
+                return items;
+            }
+        }
+    }
+
+    @Override
+    public List<Item> getLimitItems(Connection connection, ItemPagination pagination) throws SQLException {
+        try (PreparedStatement preparedStatement = connection.prepareCall(requestProperties.getSqlRequestSelectLimitItems())) {
+            Integer startDocument = pagination.getCurrentPage() * pagination.getItemsPerPage() - pagination.getItemsPerPage();
+            preparedStatement.setInt(1, startDocument);
+            preparedStatement.setInt(2, pagination.getItemsPerPage());
+            try (ResultSet resultSet = preparedStatement.executeQuery()) {
+                List<Item> items = new ArrayList<>();
+                while (resultSet.next()) {
+                    Item item = getItem(resultSet);
+                    items.add(item);
+                }
+                return items;
+            }
+        }
+    }
+
+    @Override
+    public Item getEntityByUUID(Connection connection, String uuid) throws SQLException {
+        try (PreparedStatement preparedStatement = connection.prepareCall(requestProperties.getSqlRequestSelectItemByUUID())) {
+            preparedStatement.setString(1, uuid);
+            try (ResultSet resultSet = preparedStatement.executeQuery()) {
+                Item item = null;
+                if (resultSet.next()) {
+                    item = getItem(resultSet);
+                }
+                return item;
+            }
+        }
+    }
+
+    @Override
+    public List<Item> getEntityByItemStatus(Connection connection, ItemStatus itemStatus) throws SQLException {
+        try (PreparedStatement preparedStatement = connection.prepareCall(requestProperties.getSqlRequestSelectItemByStatus())) {
+            preparedStatement.setString(1, itemStatus.name());
+            try (ResultSet resultSet = preparedStatement.executeQuery()) {
+                List<Item> items = new ArrayList<>();
+                while (resultSet.next()) {
+                    Item item = getItem(resultSet);
+                    items.add(item);
+                }
+                return items;
+            }
+        }
+    }
+
+    @Override
+    public List<Item> getLimitItemsByItemStatus(Connection connection, ItemStatus itemStatus, ItemPagination pagination) throws SQLException {
+        try (PreparedStatement preparedStatement = connection.prepareCall(requestProperties.getSqlRequestSelectLimitItemsByStatus())) {
```

Comment:
Better to separate requestProperties.getSqlRequestSelectLimitItemsByStatus() to new line